free5gc-smf-org-example/packagevariantset.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       - foo-03
     template:
       downstream:
-        packageExpr: "packageName + '-' + repository.labels['env']"
+        packageExpr: "target.package"
```

Comment:
This works - I meant for it to be `"target.package + '-' repository.labels['env']"`, but for example purposes this is just fine too.